ocs_ci/ocs/resources/pvc.py

```diff
@@ -74,6 +74,28 @@ def backed_pv_obj(self):
         pv_obj.reload()
         return pv_obj
 
+    def delete(self, wait=True, force=False, validate_pv_deleted=False):
+        """
+        Delete the PVC
+
+        Args:
+            wait (bool): True for wait for PVC deletion, False otherwise
+            force (bool): True for force deletion with --grace-period=0,
+                False otherwise
+            validate_pv_deleted (bool): True for respective PV deletion
+                validation, False otherwise
+
+        Returns:
+            dict: Dictionary represents a returned yaml file
+        """
+        backed_pv = None
+        if validate_pv_deleted:
+            backed_pv = self.backed_pv_obj
+        result = super().delete(wait=wait, force=force)
+        if validate_pv_deleted:
+            backed_pv.ocp.wait_for_delete(resource_name=self.backed_pv)
```

Comment:
How this will behave when the reclaim policy is to retain?
I think we need to figure this out

Reply:
@RazTamir yes, exactly, will have to find a way to sync between the two.
But even if we find a way to do pv deletion validation only in case the reclaim policy is delete, in some places, we'll won't to skip the validation, even if the policy is to delete. For instance in disruptive scenarios, in which we don't want to wait for PVC and PV deletion to complete

Reply:
Ok,

Let's just ensure this will not be stuck anything because the reclaim policy is to retain and it will fail on timeout

Reply:
the default is False, we'll have to be careful when changing to True